frontend/src/store/episode/actions.ts

```diff
@@ -5,15 +5,24 @@ import {
   Comment,
   CommentFormCreatePayload,
   AsyncThunkConfig,
+  Podcast,
 } from 'common/types/types';
 import { ActionType } from './common';
 
-const loadEpisode = createAsyncThunk<Episode, number, AsyncThunkConfig>
+type PodcastAndEpisode = {
```

Comment:
```suggestion
type LoadEpisodePayload = {
```
move it to `frontend/src/store/episode/common`